modules/xmpp/strophe.ping.js

```diff
@@ -23,6 +23,19 @@ const PING_TIMEOUT = 15000;
  */
 const PING_THRESHOLD = 3;
 
+/**
+ * The number of timestamps of send pings to keep.
+ * @type {number} number of timestamps.
+ */
+const PING_TIMESTAMPS_TO_KEEP = 12;
+
+/**
+ * If execution of pings is higher than this threshold, we consider that the
+ * ping was suspended.
+ * @type {number}
+ */
+const SUSPEND_INTERVAL_THRESHOLD = PING_TIMEOUT;
```

Comment:
What is the relationship between SUSPEND_INTERVAL_THRESHOLD and PING_TIMEOUT? 

Wouldn't we want SUSPEND_INTERVAL_THRESHOLD to depend on PING_INTERVAL instead (if PING_INTERVAL is 1000, we could detect suspension sooner; if it is 100_000 we would have false positive detections)?

Reply:
Yep, I was thinking to make them dependent ... but if we are always sending the interval maybe I will remove this threshold and we detect this on the analytics side of things and move the threshold there.